common/scripted_triggers/iisc_triggers.txt

```diff
@@ -91,22 +102,22 @@ iisc_should_build_builder = {
 		which = iisc_budget_builder
 		value >= @iisc_builder_cost
 	}
+	iisc_has_not_enough_builders = yes
+}
+
+# this = corp
+iisc_has_not_enough_builders = {
 	count_owned_ship = {
 		limit = {
 			is_ship_size = constructor
 		}
-		count < 1		# TODO change
+		count < 5
 	}
 }
 
 # this = corp
 iisc_should_destroy_builder = {
-	count_owned_ship = {
-		limit = {
-			is_ship_size = constructor
-		}
-		count > 5		# TODO change
-	}
+	always = no
```

Comment:
And also like above, once there's not much for the corp to construct we should probably be reducing their number of ships flying around.